README.md

```diff
@@ -19,6 +19,7 @@ Pick a building block API (for example, pub-sub, state management) and rapidly t
 | Actors | Coming soon... |
 | [Configuration](./configuration) | Get configuration items as key/value pairs or subscribe to changes whenever a configuration item changes |
 | [Resiliency](./resiliency) | Define and apply fault-tolerant policies (retries/back-offs, timeouts and circuit breakers) to your Dapr API requests |
+| [Workflow](./workflows) | Coordinated, possibly long running activities in an application that are resilient to failures  |
```

Comment:
I suggest that we align this with the wording in the quickstart docs here like 
this. Something like this

"Dapr Workflow makes orchestrating the logic required for messaging, state management, and failure handling across various microservices easier for developers. Dapr Workflow enable you to create long running, fault-tolerant, stateful applications."

or shorten to 

"Dapr Workflow enable you to create long running, fault-tolerant, stateful applications."